### PR TITLE
OBSDOCS-1242 Distributed tracing UI plugin

### PR DESCRIPTION
**Pull request by gabriel-rh**
Version(s):
4.12+

Issue:
[OBSDOCS-1242](https://issues.redhat.com//browse/OBSDOCS-1242)

Link to docs preview:
https://81424--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/cluster_observability_operator/ui_plugins/distributed-tracing-ui-plugin.html

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/coo-distributed-tracing-ui-plugin-using.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+
+// * observability/cluster_observability_operator/ui_plugins/distributed-tracing-ui-plugin.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="coo-distributed-tracing-ui-plugin-using_{context}"]
+= Using the {coo-full} distributed tracing UI plugin
```
[the remark this answers is not in the document]
Comment (gabriel-rh) [reply modules/coo-distributed-tracing-ui-plugin-using.adoc:7]: Distributed tracing is used a generic term here, not as a product name

File: observability/cluster_observability_operator/ui_plugins/distributed-tracing-ui-plugin.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+= Distributed tracing UI plugin
+include::_attributes/common-attributes.adoc[]
+:context: distributed-tracing-ui-plugin
+
+toc::[]
+
+:FeatureName: The Cluster Observability Operator
+include::snippets/technology-preview.adoc[leveloffset=+2]
+
+The distributed tracing UI plugin adds tracing-related features to the Administrator perspective of the OpenShift web console under **Observe** → **Traces**. You can follow requests through the front end and into the backend of microservices, helping you identify code errors and performance bottlenecks in distributed systems.
```
[the remark this answers is not in the document]
Comment (gabriel-rh) [reply observability/cluster_observability_operator/ui_plugins/distributed-tracing-ui-plugin.adoc:12]: Distributed tracing is used a generic term here, not as a product name

File: modules/coo-distributed-tracing-ui-plugin-install.adoc [hunk 3]
```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+
+// * observability/cluster_observability_operator/ui_plugins/distributed-tracing-ui-plugin.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="coo-distributed-tracing-ui-plugin-install-_{context}"]
+= Installing the {coo-full} distributed tracing UI plugin
```
[the remark this answers is not in the document]
Comment (GroceryBoyJr) [reply modules/coo-distributed-tracing-ui-plugin-install.adoc:7]: Thank you for this note, @gabriel-rh !

File: observability/cluster_observability_operator/ui_plugins/distributed-tracing-ui-plugin.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (GroceryBoyJr) [reply observability/cluster_observability_operator/ui_plugins/distributed-tracing-ui-plugin.adoc:12]: Agreed! Also agreed with some of the other quibbles vale had that you rejected in this file.

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Review by GroceryBoyJr — APPROVED** [review]
LGTM

File: modules/coo-distributed-tracing-ui-plugin-using.adoc [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+* You have logged in to the {product-title} web console.
+* You have installed the {coo-full}.
+* You have a `TempoStack` or `TempoMonolithic` multi-tenant instance in the cluster.
+
+.Procedure
+
+. In the Administrator perspective of the {product-title} web console, click **Observe** → **Traces**.
+. Select a `TempoStack` or `TempoMonolithic` instance and set a time range and query for the traces to be loaded.
```
Comment (jgbernalp) [inline modules/coo-distributed-tracing-ui-plugin-using.adoc:19]: @gabriel-rh only `TempoStack` with multi-tenancy enabled are shown in the dropdown

Comment (gabriel-rh) [reply modules/coo-distributed-tracing-ui-plugin-using.adoc]: Done

**Comment by jgbernalp** [comment]
LGTM

**Comment by lihongyan1** [comment]
@IshwarKanse Please help take a look

File: modules/coo-distributed-tracing-ui-plugin-using.adoc [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+:_mod-docs-content-type: PROCEDURE
+[id="coo-distributed-tracing-ui-plugin-using_{context}"]
+= Using the {coo-full} distributed tracing UI plugin
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+* You have logged in to the {product-title} web console.
+* You have installed the {coo-full}.
```
Comment (lihongyan1) [inline modules/coo-distributed-tracing-ui-plugin-using.adoc:13]: Suppose should be 
You have installed the {coo-full} distributed tracing UI plugin

File: modules/coo-distributed-tracing-ui-plugin-using.adoc [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+* You have a `TempoStack` multi-tenant instance in the cluster.
```
Comment (IshwarKanse) [inline modules/coo-distributed-tracing-ui-plugin-using.adoc:14]: TempoStack or TempoMonolithic instance.

File: modules/coo-distributed-tracing-ui-plugin-using.adoc [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+.Procedure
+
+. In the Administrator perspective of the {product-title} web console, click **Observe** → **Traces**.
+. Select a `TempoStack` multi-tenant instance and set a time range and query for the traces to be loaded.
```
Comment (IshwarKanse) [inline modules/coo-distributed-tracing-ui-plugin-using.adoc:19]: TempoStack or TempoMonolithc

File: modules/coo-distributed-tracing-ui-plugin-using.adoc (hunk 6 above)
Comment (gabriel-rh) [reply modules/coo-distributed-tracing-ui-plugin-using.adoc]: Done

File: modules/coo-distributed-tracing-ui-plugin-using.adoc (hunk 7 above)
Comment (gabriel-rh) [reply modules/coo-distributed-tracing-ui-plugin-using.adoc]: Done

File: observability/cluster_observability_operator/ui_plugins/distributed-tracing-ui-plugin.adoc [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+= Distributed tracing UI plugin
+include::_attributes/common-attributes.adoc[]
+:context: distributed-tracing-ui-plugin
+
+toc::[]
+
+:FeatureName: The Cluster Observability Operator
+include::snippets/technology-preview.adoc[leveloffset=+2]
+
+The distributed tracing UI plugin adds tracing-related features to the Administrator perspective of the OpenShift web console at **Observe** → **Traces**. You can follow requests through the front end and into the backend of microservices, helping you identify code errors and performance bottlenecks in distributed systems.
```
[the remark this answers is not in the document]
Comment (gabriel-rh) [reply observability/cluster_observability_operator/ui_plugins/distributed-tracing-ui-plugin.adoc:12]: generic term, not product name

File: modules/coo-distributed-tracing-ui-plugin-using.adoc [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+:_mod-docs-content-type: PROCEDURE
+[id="coo-distributed-tracing-ui-plugin-using_{context}"]
+= Using the {coo-full} distributed tracing UI plugin
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+* You have logged in to the {product-title} web console.
+* You have installed the {coo-full}.
+* You have installed the {coo-full} distributed tracing UI plugin.
```
[the remark this answers is not in the document]
Comment (gabriel-rh) [reply modules/coo-distributed-tracing-ui-plugin-using.adoc:14]: generic term, not product name

**Pull request merged**